mkdocs.yml

```
@@ -132,7 +136,7 @@ plugins:
       redirect_maps:
         "01-introduction.md": "introduction.md"
         "02-common-principles.md": "common-principles.md"
-        "03-modality-agnostic-files.md": "modality-agnostic-files.md"
+        "03-modality-agnostic-files.md": "modality-agnostic-files/dataset-description.md"
```

I think we need to keep old and add the unnumbered one, so

```suggestion
        "03-modality-agnostic-files.md": "modality-agnostic-files/dataset-description.md"
        "modality-agnostic-files.md": "modality-agnostic-files/dataset-description.md"
```

because currently we have 

https://bids-specification.readthedocs.io/en/stable/modality-agnostic-files.html

so it is the one to redirect too.  And indeed then to complement all those section redirects appropriately after 
- https://github.com/mkdocs/mkdocs-redirects/pull/73
is released